enhancements/microshift/default-csi-plugin.md

```diff
@@ -0,0 +1,668 @@
+   logical description for the deployment, and how it handles the unique aspects
+   of the platform. Aim for single topic PRs to keep discussions focused. If you
+   disagree with what is already in a document, open a new PR with suggested
+   changes.
+1. **Keep all required headers.** If a section does not apply to an
+   enhancement, explain why but do not remove the section. This part
+   of the process is enforced by the linter CI job.
+--->
+# MicroShift Default CSI Plugin
+
+<!---
+This is the title of the enhancement. Keep it simple and descriptive. A good
+title can help communicate what the enhancement is and should be considered as
+part of any review.
+
+The YAML `title` should be lowercased and spaces/punctuation should be
+replaced with `-`.
+
+The `Metadata` section above is intended to support the creation of tooling
+around the enhancement process.
+--->
+
+## Summary
+<!---
+The `Summary` section is incredibly important for producing high quality
+user-focused documentation such as release notes or a development roadmap. It
+should be possible to collect this information before implementation begins in
+order to avoid requiring implementors to split their attention between writing
+release notes and implementing the feature itself.
+
+A good summary is no more than one paragraph in length. More detail
+should go into the following sections.
+--->
+
+This enhancement proposes the adoption of a
+default [Container Storage Interface(CSI) Plugin](https://kubernetes-csi.github.io/docs/)for Microshift. For more
+details on MicroShift, see the respective [proposal](./kubernetes-for-device-edge.md). Kubernetes' CSI implementation
+provides a standardized model for exposing block and file storage systems to container workloads.
+
+[TopoLVM](https://github.com/red-hat-storage/topolvm) (downstream, ODF-LVM), is the operand of the
+[LVM-Operator](https://github.com/red-hat-storage/lvm-operator),and is proposed as MicroShift's default supported
+CSI plugin. The storage solution is maintained downstream by the OpenShift Data Foundation(ODF), and is supported on
+Single Node OpenShift (SNO). The plugin is compatible with MicroShift's recommended OS's(RHEL/Centos/Fedora).
+
+## Motivation
+<!---
+This section is for explicitly listing the motivation, goals and non-goals of
+this proposal. Describe why the change is important and the benefits to users.
+--->
+
+For proof of concept, the KubeVirt Hostpath Provisioner (HPP) was chosen smooth the ramp-up of MicroShift development
+but was not intended for use in production. MicroShift has since matured enough to warrant a storage provisioner that
+will work out of the box and be compatible with a standard RHEL 8.X / RHEL for Edge host.
+
+ODF-LVM is production-ready, enables isolation of workload and system storage, and is already supported by ODF on Single
+Node OpenShift.  ODF-LVM implements features not present in HPP, such as volume snapshotting and thin-provisioning.
```

Comment:
I would also move this paragraph down to the Proposal section. That leaves a clear motivation for making a change here and puts the details of why we are making the choice we're making together in 1 place below.